lib/src/iterable_extensions.dart

```diff
@@ -592,6 +592,8 @@ extension IterableNullableExtension<T extends Object> on Iterable<T?> {
 /// since doubles require special handling of [double.nan].
 extension IterableNumberExtension on Iterable<num> {
   /// A minimal element of the iterable, or `null` it the iterable is empty.
+  ///
+  /// If an element is NaN, it is considered to be a minimal element.
```

Comment:
I'd phrase this as:
```dart
/// If any element is [NaN](double.nan), the result is NaN.
```
It's not considering NaN to be minimal, rather it's about not even being able to answer the question of which value is minimal if one of the values is not a number. That's the general rule of "if any operand is NaN, the result is NaN" that applies to all double operations.

Ditto everywhere below.

Reply:
Ah, good point. Thank you for the explanation :+1: 
I pushed the changes.